### ISSUE TITLE
Route overview should contain distances

### ISSUE THREAD

**Issue by Framstag** [janbar/osmin#13]
I do like the route overview before one starts a route/navigation. I miss though distances. Another question: Can I reopen the overview during routing (with "cursor" on the current step)?

**Comment by janbar** [comment]
Yes you can review the overview: click the "burger menu" then choose "Navigate": it will show you the overview of the current route.
About the current step, I will check if the routing module returns the step number of the current step. If yes , then we can.

**Comment by janbar** [comment]
Also, the overview contains the distance of the route. But not for each step. To be checked too ...

**Comment by Framstag** [comment]
If the router reroutes the overview should be updated, too.

In concrete: I used the overview at the beginning to validate the route and check how in principle the route runs. For this distance for each step helps (route = drive to the motorway - long distance on motorway -  fiddle your way to through the target city).

There are likely similar expectation during driving (especially for bike and foot, but also for car).

Possible one should also show the 2 next step (but displayed smaller)

**Comment by janbar** [comment]
I made change in libosmscout to get missed data of route step. I introduce the route overview in navigator info box. By clicking the box it take full size and show you the overview. The overview move automatically in the list on the current next step. Also this step is highlighted.
I need to make some refactor around the routing, but the target is better again !

![Capture d’écran du 2021-04-06 11-18-22](https://user-images.githubusercontent.com/2353635/113688895-05cfd980-96ca-11eb-8954-b7949c60900f.png)

**Comment by janbar** [comment]
Tested on road. I will make a new release at the end of the day.

**Comment by Framstag** [comment]
Cool...but can we have deltas instead of absolute values ;-)?

IMHO this is a classical case of "different thinking", some people prefer absolute values, some might prefer relative values. To you already have a conceptional upper limit for how many options you want to add to our app ;-)?

**Comment by janbar** [comment]
Hi @Framstag , I will add a new action "Long Tap" to update the setting for delta.
Adding that in settings page could be confused for the end-user (I am), and it should be better to change this setting directly on the overview. Like that you could change the view as you need without browse the menu.
